Startup fails when docker is not running on localhost

This is caused by https://github.com/exasol/exasol-test-setup-abstraction-java/issues/33 and fixed in https://github.com/exasol/exasol-test-setup-abstraction-java/releases/tag/0.3.1. To fix this we need to upgrade to the latest version.

```
java.lang.IllegalStateException: E-ETAJ-12: Failed to ssh to the exasol database. This is required for redirecting a host port. Known mitigations:
* Make sure the database is reachable (port 22 open)
* Make sure 'ec2-user' can login to the database using your ssh-key.
        at com.exasol.exasoltestsetup.SshConnection.createSession(SshConnection.java:49)
        at com.exasol.exasoltestsetup.SshConnection.<init>(SshConnection.java:30)
        at com.exasol.exasoltestsetup.testcontainers.ExasolTestcontainerTestSetup.<init>(ExasolTestcontainerTestSetup.java:41)
        at com.exasol.exasoltestsetup.ExasolTestSetupFactory.getTestSetup(ExasolTestSetupFactory.java:48)
        at com.exasol.testsetupabstraction.server.Main.main(Main.java:16)
Caused by: com.jcraft.jsch.JSchException: java.net.ConnectException: Connection refused (Connection refused)
        at com.jcraft.jsch.Util.createSocket(Util.java:394)
        at com.jcraft.jsch.Session.connect(Session.java:215)
        at com.exasol.exasoltestsetup.SshConnection.createSession(SshConnection.java:38)
        ... 4 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
        at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
        at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
        at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
        at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
        at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.base/java.net.Socket.connect(Socket.java:609)
        at java.base/java.net.Socket.connect(Socket.java:558)
        at java.base/java.net.Socket.<init>(Socket.java:454)
        at java.base/java.net.Socket.<init>(Socket.java:231)
        at com.jcraft.jsch.Util$1.run(Util.java:362)
        at java.base/java.lang.Thread.run(Thread.java:829)

panic: failed to start test-setup-abstraction server. Cause: exit status 100
```

--- a/.github/workflows/broken_links_checker.yml
+++ b/.github/workflows/broken_links_checker.yml
@@ -2,7 +2,7 @@ name: Broken Links Checker
 
 on:
   schedule:
-    - cron: "0 5 * * *"
+    - cron: "0 5 * * 0"
   push:
     branches:
       - main
@@ -19,8 +19,8 @@ jobs:
       - name: Configure broken links checker
         run: |
           mkdir -p ./target
-          echo '{ "aliveStatusCodes": [429, 200] }' > ./target/broken_links_checker.json
-      - uses: gaurav-nelson/github-action-markdown-link-check@1.0.13
+          echo '{ "aliveStatusCodes": [429, 200], "ignorePatterns": [{"pattern": "^https?://(www.)?opensource.org"}] }' > ./target/broken_links_checker.json
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
         with:
           use-quiet-mode: 'yes'
           use-verbose-mode: 'yes'

--- a/build.sh
+++ b/build.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -eo pipefail
-pk_version=2.4.3
+pk_version=2.4.4
 mvn org.apache.maven.plugins:maven-dependency-plugin:3.1.1:get "-Dartifact=com.exasol:project-keeper-cli:$pk_version:jar"
 java -jar "$HOME/.m2/repository/com/exasol/project-keeper-cli/$pk_version/project-keeper-cli-$pk_version.jar" verify
 mvn --file server/ install

--- a/dependencies.md
+++ b/dependencies.md
@@ -3,79 +3,84 @@
 
 ## Compile Dependencies
 
-| Dependency                              | License                                       |
-| --------------------------------------- | --------------------------------------------- |
-| [exasol-test-setup-abstraction-java][0] | [MIT][1]                                      |
-| [javalin][2]                            | [The Apache Software License, Version 2.0][3] |
+| Dependency                                          | License                                       |
+| --------------------------------------------------- | --------------------------------------------- |
+| [exasol-test-setup-abstraction-java][0]             | [MIT][1]                                      |
+| [javalin][2]                                        | [The Apache Software License, Version 2.0][3] |
+| [AWS Java SDK :: Services :: AWS CloudFormation][4] | [Apache License, Version 2.0][5]              |
 
 ## Test Dependencies
 
 | Dependency                     | License                           |
 | ------------------------------ | --------------------------------- |
-| [JUnit Jupiter Engine][4]      | [Eclipse Public License v2.0][5]  |
-| [JUnit Jupiter Params][4]      | [Eclipse Public License v2.0][5]  |
-| [Hamcrest][8]                  | [BSD License 3][9]                |
-| [mockito-junit-jupiter][10]    | [The MIT License][11]             |
-| [REST Assured][12]             | [Apache 2.0][13]                  |
-| [JUnit5 System Extensions][14] | [Eclipse Public License v2.0][15] |
+| [JUnit Jupiter Engine][6]      | [Eclipse Public License v2.0][7]  |
+| [JUnit Jupiter Params][6]      | [Eclipse Public License v2.0][7]  |
+| [Hamcrest][10]                 | [BSD License 3][11]               |
+| [mockito-junit-jupiter][12]    | [The MIT License][13]             |
+| [REST Assured][14]             | [Apache 2.0][15]                  |
+| [Apache Commons Codec][16]     | [Apache License, Version 2.0][17] |
+| [JUnit5 System Extensions][18] | [Eclipse Public License v2.0][19] |
 
 ## Plugin Dependencies
 
 | Dependency                                              | License                                       |
 | ------------------------------------------------------- | --------------------------------------------- |
-| [SonarQube Scanner for Maven][16]                       | [GNU LGPL 3][17]                              |
-| [Apache Maven Compiler Plugin][18]                      | [Apache License, Version 2.0][19]             |
-| [Apache Maven Enforcer Plugin][20]                      | [Apache License, Version 2.0][19]             |
-| [Maven Flatten Plugin][22]                              | [Apache Software Licenese][3]                 |
-| [org.sonatype.ossindex.maven:ossindex-maven-plugin][24] | [ASL2][3]                                     |
-| [Reproducible Build Maven Plugin][26]                   | [Apache 2.0][3]                               |
-| [Maven Surefire Plugin][28]                             | [Apache License, Version 2.0][19]             |
-| [Versions Maven Plugin][30]                             | [Apache License, Version 2.0][19]             |
-| [Apache Maven Assembly Plugin][32]                      | [Apache License, Version 2.0][19]             |
-| [Apache Maven JAR Plugin][34]                           | [Apache License, Version 2.0][19]             |
-| [Artifact reference checker and unifier][36]            | [MIT][1]                                      |
-| [JaCoCo :: Maven Plugin][38]                            | [Eclipse Public License 2.0][39]              |
-| [error-code-crawler-maven-plugin][40]                   | [MIT][1]                                      |
-| [Exec Maven Plugin][42]                                 | [Apache License 2][3]                         |
-| [Maven Clean Plugin][44]                                | [The Apache Software License, Version 2.0][3] |
-| [Maven Resources Plugin][46]                            | [The Apache Software License, Version 2.0][3] |
-| [Maven Install Plugin][48]                              | [The Apache Software License, Version 2.0][3] |
-| [Maven Deploy Plugin][50]                               | [The Apache Software License, Version 2.0][3] |
-| [Maven Site Plugin 3][52]                               | [The Apache Software License, Version 2.0][3] |
+| [SonarQube Scanner for Maven][20]                       | [GNU LGPL 3][21]                              |
+| [Apache Maven Compiler Plugin][22]                      | [Apache License, Version 2.0][17]             |
+| [Apache Maven Enforcer Plugin][24]                      | [Apache License, Version 2.0][17]             |
+| [Maven Flatten Plugin][26]                              | [Apache Software Licenese][3]                 |
+| [Exec Maven Plugin][28]                                 | [Apache License 2][3]                         |
+| [org.sonatype.ossindex.maven:ossindex-maven-plugin][30] | [ASL2][3]                                     |
+| [Reproducible Build Maven Plugin][32]                   | [Apache 2.0][3]                               |
+| [Maven Surefire Plugin][34]                             | [Apache License, Version 2.0][17]             |
+| [Versions Maven Plugin][36]                             | [Apache License, Version 2.0][17]             |
+| [Apache Maven Assembly Plugin][38]                      | [Apache License, Version 2.0][17]             |
+| [Apache Maven JAR Plugin][40]                           | [Apache License, Version 2.0][17]             |
+| [Artifact reference checker and unifier][42]            | [MIT][1]                                      |
+| [JaCoCo :: Maven Plugin][44]                            | [Eclipse Public License 2.0][45]              |
+| [error-code-crawler-maven-plugin][46]                   | [MIT][1]                                      |
+| [Maven Clean Plugin][48]                                | [The Apache Software License, Version 2.0][3] |
+| [Maven Resources Plugin][50]                            | [The Apache Software License, Version 2.0][3] |
+| [Maven Install Plugin][52]                              | [The Apache Software License, Version 2.0][3] |
+| [Maven Deploy Plugin][54]                               | [The Apache Software License, Version 2.0][3] |
+| [Maven Site Plugin 3][56]                               | [The Apache Software License, Version 2.0][3] |
 
-[15]: http://www.eclipse.org/legal/epl-v20.html
+[19]: http://www.eclipse.org/legal/epl-v20.html
 [3]: http://www.apache.org/licenses/LICENSE-2.0.txt
-[28]: https://maven.apache.org/surefire/maven-surefire-plugin/
-[13]: http://www.apache.org/licenses/LICENSE-2.0.html
-[44]: http://maven.apache.org/plugins/maven-clean-plugin/
+[34]: https://maven.apache.org/surefire/maven-surefire-plugin/
+[15]: http://www.apache.org/licenses/LICENSE-2.0.html
+[48]: http://maven.apache.org/plugins/maven-clean-plugin/
+[4]: https://aws.amazon.com/sdkforjava
 [1]: https://opensource.org/licenses/MIT
-[10]: https://github.com/mockito/mockito
-[22]: https://www.mojohaus.org/flatten-maven-plugin/
-[42]: http://www.mojohaus.org/exec-maven-plugin
-[30]: http://www.mojohaus.org/versions-maven-plugin/
-[9]: http://opensource.org/licenses/BSD-3-Clause
-[18]: https://maven.apache.org/plugins/maven-compiler-plugin/
+[12]: https://github.com/mockito/mockito
+[26]: https://www.mojohaus.org/flatten-maven-plugin/
+[28]: http://www.mojohaus.org/exec-maven-plugin
+[16]: https://commons.apache.org/proper/commons-codec/
+[36]: http://www.mojohaus.org/versions-maven-plugin/
+[11]: http://opensource.org/licenses/BSD-3-Clause
+[22]: https://maven.apache.org/plugins/maven-compiler-plugin/
 [0]: https://github.com/exasol/exasol-test-setup-abstraction-java/
-[39]: https://www.eclipse.org/legal/epl-2.0/
-[17]: http://www.gnu.org/licenses/lgpl.txt
-[38]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
-[12]: http://code.google.com/p/rest-assured
-[11]: https://github.com/mockito/mockito/blob/main/LICENSE
-[26]: http://zlika.github.io/reproducible-build-maven-plugin
-[16]: http://sonarsource.github.io/sonar-scanner-maven/
-[19]: https://www.apache.org/licenses/LICENSE-2.0.txt
-[20]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
-[5]: https://www.eclipse.org/legal/epl-v20.html
-[48]: http://maven.apache.org/plugins/maven-install-plugin/
-[4]: https://junit.org/junit5/
-[24]: https://sonatype.github.io/ossindex-maven/maven-plugin/
-[14]: https://github.com/itsallcode/junit5-system-extensions
+[45]: https://www.eclipse.org/legal/epl-2.0/
+[21]: http://www.gnu.org/licenses/lgpl.txt
+[44]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
+[5]: https://aws.amazon.com/apache2.0
+[14]: http://code.google.com/p/rest-assured
+[13]: https://github.com/mockito/mockito/blob/main/LICENSE
+[32]: http://zlika.github.io/reproducible-build-maven-plugin
+[17]: https://www.apache.org/licenses/LICENSE-2.0.txt
+[20]: http://sonarsource.github.io/sonar-scanner-maven/
+[24]: https://maven.apache.org/enforcer/maven-enforcer-plugin/
+[7]: https://www.eclipse.org/legal/epl-v20.html
+[52]: http://maven.apache.org/plugins/maven-install-plugin/
+[6]: https://junit.org/junit5/
+[30]: https://sonatype.github.io/ossindex-maven/maven-plugin/
+[18]: https://github.com/itsallcode/junit5-system-extensions
 [2]: https://javalin.io/
-[8]: http://hamcrest.org/JavaHamcrest/
-[50]: http://maven.apache.org/plugins/maven-deploy-plugin/
-[52]: http://maven.apache.org/plugins/maven-site-plugin/
-[46]: http://maven.apache.org/plugins/maven-resources-plugin/
-[36]: https://github.com/exasol/artifact-reference-checker-maven-plugin
-[40]: https://github.com/exasol/error-code-crawler-maven-plugin
-[34]: https://maven.apache.org/plugins/maven-jar-plugin/
-[32]: https://maven.apache.org/plugins/maven-assembly-plugin/
+[10]: http://hamcrest.org/JavaHamcrest/
+[54]: http://maven.apache.org/plugins/maven-deploy-plugin/
+[56]: http://maven.apache.org/plugins/maven-site-plugin/
+[50]: http://maven.apache.org/plugins/maven-resources-plugin/
+[42]: https://github.com/exasol/artifact-reference-checker-maven-plugin
+[46]: https://github.com/exasol/error-code-crawler-maven-plugin
+[40]: https://maven.apache.org/plugins/maven-jar-plugin/
+[38]: https://maven.apache.org/plugins/maven-assembly-plugin/

--- a/dependencies.md
+++ b/dependencies.md
@@ -55,6 +55,7 @@
 [30]: http://www.mojohaus.org/versions-maven-plugin/
 [9]: http://opensource.org/licenses/BSD-3-Clause
 [18]: https://maven.apache.org/plugins/maven-compiler-plugin/
+[0]: https://github.com/exasol/exasol-test-setup-abstraction-java/
 [39]: https://www.eclipse.org/legal/epl-2.0/
 [17]: http://www.gnu.org/licenses/lgpl.txt
 [38]: https://www.jacoco.org/jacoco/trunk/doc/maven.html
@@ -76,6 +77,5 @@
 [46]: http://maven.apache.org/plugins/maven-resources-plugin/
 [36]: https://github.com/exasol/artifact-reference-checker-maven-plugin
 [40]: https://github.com/exasol/error-code-crawler-maven-plugin
-[0]: https://github.com/exasol/exasol-test-setup-abstraction-java
 [34]: https://maven.apache.org/plugins/maven-jar-plugin/
 [32]: https://maven.apache.org/plugins/maven-assembly-plugin/

--- a/doc/changes/changelog.md
+++ b/doc/changes/changelog.md
@@ -1,3 +1,4 @@
 # Changes
 
+* [0.1.1](changes_0.1.1.md)
 * [0.1.0](changes_0.1.0.md)

--- a/doc/changes/changes_0.1.1.md
+++ b/doc/changes/changes_0.1.1.md
@@ -13,3 +13,10 @@ Code name: 0.1.1: Fixed failing startup when docker is not running on localhost
 ### Compile Dependency Updates
 
 * Updated `com.exasol:exasol-test-setup-abstraction-java:0.3.0` to `0.3.1`
+* Updated `io.javalin:javalin:4.5.0` to `4.6.0`
+* Added `software.amazon.awssdk:cloudformation:2.17.198`
+
+### Test Dependency Updates
+
+* Added `commons-codec:commons-codec:1.15`
+* Updated `io.rest-assured:rest-assured:5.0.1` to `5.1.0`

--- a/doc/changes/changes_0.1.1.md
+++ b/doc/changes/changes_0.1.1.md
@@ -13,4 +13,3 @@ Code name: 0.1.1: Fixed failing startup when docker is not running on localhost
 ### Compile Dependency Updates
 
 * Updated `com.exasol:exasol-test-setup-abstraction-java:0.3.0` to `0.3.1`
-* Updated `io.javalin:javalin:4.5.0` to `4.6.0`

--- a/doc/changes/changes_0.1.1.md
+++ b/doc/changes/changes_0.1.1.md
@@ -1,0 +1,16 @@
+# Server for the Exasol Test-Setup Abstraction 0.1.1, released 2022-05-24
+
+Code name: 0.1.1: Fixed failing startup when docker is not running on localhost
+
+## Summary
+
+## Features
+
+* #5: Fixed failing startup when docker is not running on localhost
+
+## Dependency Updates
+
+### Compile Dependency Updates
+
+* Updated `com.exasol:exasol-test-setup-abstraction-java:0.3.0` to `0.3.1`
+* Updated `io.javalin:javalin:4.5.0` to `4.6.0`

--- a/go-client/TestSetupAbstraction.go
+++ b/go-client/TestSetupAbstraction.go
@@ -28,7 +28,7 @@ type TestSetupAbstraction struct {
 	stoppedMutex   *sync.Mutex
 }
 
-const serverVersion = "0.1.0"
+const serverVersion = "0.1.1"
 const serverJar = "exasol-test-setup-abstraction-server-" + serverVersion + ".jar"
 
 func Create(configFilePath string) TestSetupAbstraction {

--- a/go-client/TestSetupAbstraction_test.go
+++ b/go-client/TestSetupAbstraction_test.go
@@ -1,10 +1,10 @@
 package exasol_test_setup_abstraction_go
 
 import (
+	"os"
 	"testing"
-)
 
-import (
+	"github.com/antchfx/xmlquery"
 	"github.com/stretchr/testify/suite"
 )
 
@@ -53,4 +53,17 @@ func (suite *TestSetupAbstractionSuite) TestMakeTcpServiceAccessibleFromDatabase
 	serviceAddress := testSetup.MakeTcpServiceAccessibleFromDatabase(ServiceAddress{"localhost", 134})
 	suite.Assert().NotNil(serviceAddress)
 	suite.Assert().NotEmpty(serviceAddress)
+}
+
+func (suite *TestSetupAbstractionSuite) TestServerVersionCorrect() {
+	suite.Assert().Equal(suite.readVersionFromPom(), serverVersion, "version in pom.xml and constant match")
+}
+
+func (suite *TestSetupAbstractionSuite) readVersionFromPom() string {
+	pomFile, err := os.Open("../server/pom.xml")
+	suite.NoError(err)
+	defer pomFile.Close()
+	pom, err := xmlquery.Parse(pomFile)
+	suite.NoError(err)
+	return xmlquery.FindOne(pom, "/project/version").InnerText()
 }

--- a/go-client/go.mod
+++ b/go-client/go.mod
@@ -6,12 +6,18 @@ require github.com/exasol/exasol-driver-go v0.4.0
 
 require github.com/stretchr/testify v1.7.1
 
+require github.com/antchfx/xmlquery v1.3.10
+
 require (
+	github.com/antchfx/xpath v1.2.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/exasol/error-reporting-go v0.1.1 // indirect
+	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/gorilla/websocket v1.5.0 // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	golang.org/x/net v0.0.0-20220325170049-de3da57026de // indirect
+	golang.org/x/text v0.3.7 // indirect
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )

--- a/server/pk_generated_parent.pom
+++ b/server/pk_generated_parent.pom
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>exasol-test-setup-abstraction-server-generated-parent</artifactId>
-    <version>0.1.0</version>
+    <version>0.1.1</version>
     <packaging>pom</packaging>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>

--- a/server/pom.xml
+++ b/server/pom.xml
@@ -30,7 +30,7 @@
         <dependency>
             <groupId>io.javalin</groupId>
             <artifactId>javalin</artifactId>
-            <version>4.6.0</version>
+            <version>4.5.0</version>
         </dependency>
         <!-- test dependencies -->
         <dependency>

--- a/server/pom.xml
+++ b/server/pom.xml
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>exasol-test-setup-abstraction-server</artifactId>
@@ -30,7 +29,13 @@
         <dependency>
             <groupId>io.javalin</groupId>
             <artifactId>javalin</artifactId>
-            <version>4.5.0</version>
+            <version>4.6.0</version>
+        </dependency>
+        <!-- Explicitly upgrade dependency to fix vulnerability in io.netty:netty-common:4.1.72: https://ossindex.sonatype.org/vulnerability/CVE-2022-24823 -->
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>cloudformation</artifactId>
+            <version>2.17.198</version>
         </dependency>
         <!-- test dependencies -->
         <dependency>
@@ -60,7 +65,14 @@
         <dependency>
             <groupId>io.rest-assured</groupId>
             <artifactId>rest-assured</artifactId>
-            <version>5.0.1</version>
+            <version>5.1.0</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- Upgrade transitive dependency from rest-assured to fix vulnerability https://ossindex.sonatype.org/vulnerability/sonatype-2012-0050 -->
+        <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.15</version>
             <scope>test</scope>
         </dependency>
         <dependency>
@@ -103,6 +115,22 @@
                         </configuration>
                     </execution>
                 </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.sonatype.ossindex.maven</groupId>
+                <artifactId>ossindex-maven-plugin</artifactId>
+                <configuration>
+                    <excludeVulnerabilityIds>
+                        <!-- Three vulnerabilities in transitive dependency in org.apache.xmlrpc:xmlrpc-common and org.apache.xmlrpc:xmlrpc-client.
+                    These will be fixed in https://github.com/exasol/exasol-test-setup-abstraction-java/issues/38 -->
+                        <exclude>CVE-2016-5003</exclude>
+                        <exclude>CVE-2016-5002</exclude>
+                        <exclude>CVE-2016-5004</exclude>
+                        <!-- Vulnerability in io.netty:netty-handler:4.1.77.Final https://ossindex.sonatype.org/vulnerability/sonatype-2020-0026
+                    Excluded until a fixed version is available. -->
+                        <exclude>sonatype-2020-0026</exclude>
+                    </excludeVulnerabilityIds>
+                </configuration>
             </plugin>
         </plugins>
     </build>

--- a/server/pom.xml
+++ b/server/pom.xml
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>exasol-test-setup-abstraction-server</artifactId>
-    <version>0.1.0</version>
+    <version>0.1.1</version>
     <name>Server for the Exasol Test-Setup Abstraction</name>
     <description>Test setup abstraction backend for Go and other languages.</description>
     <url>https://github.com/exasol/exasol-test-setup-abstraction-server/</url>
@@ -25,12 +25,12 @@
         <dependency>
             <groupId>com.exasol</groupId>
             <artifactId>exasol-test-setup-abstraction-java</artifactId>
-            <version>0.3.0</version>
+            <version>0.3.1</version>
         </dependency>
         <dependency>
             <groupId>io.javalin</groupId>
             <artifactId>javalin</artifactId>
-            <version>4.5.0</version>
+            <version>4.6.0</version>
         </dependency>
         <!-- test dependencies -->
         <dependency>
@@ -98,9 +98,7 @@
                         <configuration>
                             <executable>${project.basedir}/installServerLocally.sh</executable>
                             <arguments>
-                                <argument>
-                                    ${project.basedir}/target/exasol-test-setup-abstraction-server-${project.version}.jar
-                                </argument>
+                                <argument>${project.basedir}/target/exasol-test-setup-abstraction-server-${project.version}.jar</argument>
                             </arguments>
                         </configuration>
                     </execution>
@@ -111,7 +109,7 @@
     <parent>
         <artifactId>exasol-test-setup-abstraction-server-generated-parent</artifactId>
         <groupId>com.exasol</groupId>
-        <version>0.1.0</version>
+        <version>0.1.1</version>
         <relativePath>pk_generated_parent.pom</relativePath>
     </parent>
 </project>

